astropy/tests/tests/test_runner.py

```diff
@@ -1,9 +1,11 @@
-from astropy.tests.runner import TestRunner, TestRunnerBase, keyword
+from astropy.tests.runner import TestRunner as T_Runner
```

Comment:
What about ``_TestRunner``? Also, add a comment so we know why this is this way.